field.go

```diff
@@ -400,6 +400,15 @@ func Object(key string, val zapcore.ObjectMarshaler) Field {
 	return Field{Key: key, Type: zapcore.ObjectMarshalerType, Interface: val}
 }
 
+// InlineObject is similar to Object, but does not nest the object under a field
+// name, but adds the fields to the current namespace inline.
+func InlineObject(val zapcore.ObjectMarshaler) Field {
```

Comment:
Is there a different name we can use here? How do you feel about `zap.Inline`?

Reply:
`Inline` is fine with me, I originally had `Object` in the name to match the interface we accept, but renamed everything to just `Inline`